NewRelic/NewRelicInteractorInterface.php

```diff
@@ -34,6 +34,14 @@ public function ignoreTransaction();
 
     /**
      * @param string $name
+     * @param array  $attributes
+     *
+     * @return void
+     */
+    public function addCustomEvent($name, array $attributes);
```

Comment:
Adding a method to an interface is a BC break.

What's about creating a new interface and checking the interface in the reponseListener and LoggerDecorator?